FHEM/lib/signalduino_protocols.hash

```diff
@@ -174,7 +176,7 @@
 			zero			=> [1,-3],
 			clockabs     	=> 500,			# -1 = auto
 			format 			=> 'twostate',	# tristate can't be migrated from bin into hex!
-			preamble		=> 'p5#',			# Append to converted message	
+			preamble		=> 'U5#',		# Append to converted message	
 			clientmodule    => 'IT',
 			modulematch     => '^i......',
```

Comment:
Das mit dem IT Modul dürfte nicht mehr Stimmen oder?

Reply:
Wenn es nicht mehr stimmt. Raus bzw auskommentieren!

Reply:
Ich denke U5# Nachrichten werden nicht an das IT Modul geleitet und der angegebene modulematch stimmt dann auch nicht. Wenn ich hier falsch liege, dann erhelle mich :)

Reply:
Ich habe mal gelesen bzw. gelesen zu haben, das Protokoll ist nur zum simulieren gewesen daher auch der Vermerk "Similar protocol as intertechno".

Ich habe es auf U gesetzt um es nicht komplett herauszunehmen. So wie die Diskussion damals lief klang es, als wäre es "überflüssig" weil eben "Similar protocol as intertechno".

Da ein kleines u ja noch verarbeitet wird und auch kein y oder p bzw m zutreffen, daher großes U um es zu "parken" vor dem auskommentieren. Wenn Ihr auch der Meinung seit es wird nicht mehr benötigt weil damals nur "Similar ..." so können wir es auskommentieren und den Platz für ein eindeutiges Protokoll schaffen ;-)

Reply:
Ich kann jetzt leider nicht rekonstruieren was damals war.

Vermutlich kam der Satz von mir und sollte ausdrücken. Ähnlich dem Intertechno Protokoll.
Die Preamble p5 und Modulematch '^i.  haben aber noch nie zusammen gepasst.

Wir können festhalten, dass diese Nachrichten wohl nie beim IT Modul angekommen sind.
Ob man es jetzt noch braucht, hmm. Dazu müsste ich mal suchen wie es dazu kam.

Reply:
Okay ich habe es rekonstruiert... die Teile liegen bei mir im Schrank.
Hersteller Unitec, Modellnummer 6899/45108   434 Mhz 

:)

Reply:
Dann bitte eine klare eindeutige Definition oder die Hardware kommt in die gelbe Tonne ;-)

Gibst du mir klare RAWMSG um die Definition zu überarbeiten oder machst du es selbst?

Reply:
War schon dabei.
Also Protokoll 5 passt nicht. Müsste also überarbeitet werden.
DIP Schalter hat meine im übrigen  nicht :)

```
2018.11.28 21:14:23.365 4: sduino/msg READredu: MU;P0=-31960;P1=660;P2=401;P3=-1749;P5=276;D=232353232323232323232323232353535353232323535353535353535353535010;CP=5;R=38;
2018.11.28 21:14:23.616 4: sduino/msg READredu: MU;P0=-1757;P1=124;P2=218;P3=282;P5=-31972;P6=644;P7=-9624;D=010201020303030202030303020303030202020202020203030303035670;CP=2;R=32;
2018.11.28 21:14:23.861 4: sduino/msg READredu: MU;P0=-1850;P1=172;P3=-136;P5=468;P6=236;D=010101010101310506010101010101010101010101010101010101010;CP=1;R=30;

```

Was machen wir jetzt hier mit dem Protokoll 5 - das ist ja eigentlich fast egal oder?

Reply:
Nagut :)
A AN:
`MU;P0=132;P1=-4680;P2=508;P3=-1775;P4=287;P6=192;D=123434343434343634343436363434343636343434363634343036363434343;CP=4;R=2;`

A AUS:
`MU;P0=-1692;P1=132;P2=194;P4=355;P5=474;P7=-31892;D=010202040505050505050404040404040404040470;CP=4;R=27;
`

Der  AN Befehl lässt sich deutlich schlechter empfangen als der AUS Befehle. Sehr seltsam.
Die Zeiten sind auch recht unterschiedlich. Vermutlich haben ich die deshalb nie integriert.

Reply:
Ich glaube ich kommentiere dein die ID 5 aus :-D
Sehr eindeutig kommt hier nichts heraus aufgrund der sehr variierenden Zeiten.

```
# MU;P0=-31960;P1=660;P2=401;P3=-1749;P5=276;D=2 32 35 32 32 32 32 32 32 32 32 32 32 32 35 35 35 35 32 32 32 35 35 35 35 35 35 35 35 35 35 35 010;CP=5;R=38;
         P2=401; - Start?
P2=401;P3=-1749;
P3=-1749;P5=276;

# MU;P0=-1757;P1=124;P2=218;P3=282;P5=-31972;P6=644;P7=-9624;D=0102 01 02 03 03 03 02 02 03 03 03 02 03 03 03 02 02 02 02 02 02 02 03 03 03 03 03 5670;CP=2;R=32;
P0=-1757;P3=282; - Start?
P0=-1757;P2=218; - 1
P0=-1757;P3=282; - 0

# MU;P0=-1850;P1=172;P3=-136;P5=468;P6=236;D=01 01 01 01 01 01 31 05 0601010101010101010101010101010101010101 0;CP=1;R=30;
P0=-1850; P5=468; - Start?
P0=-1850; P6=236; - 1
P0=-1850; P1=172; - 0

# A AN:
# MU;P0=132;P1=-4680;P2=508;P3=-1775;P4=287;P6=192;D=12 34 34343434343634343436363434343636343434363634343036363434343;CP=4;R=2;
P3=-1775;P4=287; - Start?

# A AUS:
# MU;P0=-1692;P1=132;P2=194;P4=355;P5=474;P7=-31892;D=0102 02 04 05 05 05 05 05 05 04 04 04 04 04 04 04 04 04 04 70;CP=4;R=27;
P0=-1692;P2=194; - Start?
P0=-1692;P4=355; - 1
P0=-1692;P5=474; - 0
```

Auskommentieren oder für die Jarolift ID gleich nutzen?